pkg/expansion/fixtures/fixtures.go

```diff
@@ -205,6 +222,21 @@ spec:
         kinds: []
 `
 
+	AssignImage = `
+apiVersion: mutations.gatekeeper.sh/v1alpha1
+kind: AssignImage
+metadata:
+  name: tag-v2
+spec:
+  applyTo:
+  - groups: [""]
+    kinds: ["Pod"]
+    versions: ["v1"]
+  location: "spec.containers[name:nginx].image"
```

Comment:
Is there a test for when applyTo doesn't have `spec.containers[name:nginx].image` as a valid location?

Reply:
It uses [`parser.Parse`](https://github.com/open-policy-agent/gatekeeper/blob/master/pkg/mutation/path/parser/parser.go#L33) to validate the `Location` field, which is tested separately in [`parser_test.go`](https://github.com/open-policy-agent/gatekeeper/blob/master/pkg/mutation/path/parser/parser_test.go#L26).

Specific `Location` checks for this mutator, such as requiring that the `Location` does not end with a list or contain a `metadata` root, are tested in `TestMutatorForAssignImage` (in `assignimage_mutator_test.go`).